numpy/core/defchararray.py

```diff
@@ -835,6 +844,15 @@ def isdigit(a):
     See Also
     --------
     str.isdigit
+
+    Examples
+    --------
+    >>> a = np.array(['a', 'b', '0'])
+    >>> np.char.isdigit(a)
+    array([False, False,  True])
+    >>> a = np.array([['a', 'b', '0'], ['c','1','2']])
```

Comment:
Super nit, but please add the spaces:
```suggestion
    >>> a = np.array([['a', 'b', '0'], ['c', '1', '2']])
```
to conform to PEP 8 coding style.